cfep-23.md

```diff
@@ -0,0 +1,73 @@
+- Users can "follow" threads they find interesting, which has its own dedicated view. This makes it easier to track updates to conversations you have started or participated in.
+- Private rooms are also available, but they don't offer end-to-end encryption. For this matter, a sensitive communications protocol is proposed below.
+
+## Proposal
+
+After accepting this proposal, we will implement the following decisions:
+
+- The Zulip instance published at `conda-forge.zulipchat.com` will be open to the public with world-readable permissions.
+- The General channel will be the replacement for the `conda-forge/conda-forge.github.io` chat room. This will be reflected in the conda-forge.org documentation, and reminded periodically in the Element room.
+- The conda-forge core team will move its private communications to the Zulip instance, and stop using the Element rooms. A last message will be published in the Element core chat room with the details on how to transition to the Zulip instance. 
+- New channels will be created in the instance as considered necessary (e.g. infrastructure, compilers, etc).
+- Sensitive topics will NOT be discussed directly in the Zulip instance. Instead they will follow the protocol described below.
+- Some Element rooms related to conda-forge (e.g., rooms where we interact with other communities like bioconda) will remain unchanged as needed. We will however strive to move as much of the conda-forge Element space to Zulip as possible.
+
+Three months after the approval of this CFEP, the core team will consider the migration finished and the public rooms will become private and archived. If the core team estimates that there's still sufficient activity, another three months will be granted, with periodic reminders of the new Zulip instance. After that period, no more extensions will take place.
+
+## Sensitive communications protocol
+
+Since Zulip doesn't have end-to-end encryption, it is not the best place to discuss sensitive matters. Instead, we propose the following protocol.
+
+1. Core members will announce that a new sensitive topic needs to be discussed in the Zulip channel for `core`. A small description describing the situation without revealing sensitive details should be shared.
+2. A new private room will be created in Matrix and announced in the private topic. Interested core members can voluntarily join the room to participate or observe the conversation.
+3. When the issue has been addressed, a summary of the events and decisions taken will be captured in the secret vault. The notes document should contain the date, topic title, participants and any other necessary information.
+4. Once the summary has been safely stored, a short announcement should be posted to the Zulip topic informing about the publication of the relevant notes in the vault.
+5. After a short period has passed with no comments, the Element room will be deleted and the Zulip topic will be marked as solved.
```

Comment:
Think it is worth noting that the proposed solution for handling sensitive topics relies on Matrix, which we have noted above has trouble validating keys periodically (to be clear Matrix is being overly secure here as opposed to not secure). Ideally we come up with a better solution than relying on Matrix for this

Admittedly there is not a clear alternative atm, but we should investigate moving sensitive communication off Matrix as well. This can be a follow on proposal once we have determined a clear path